### PR TITLE
[CSM-249] Rename wallet api entries

### PR DESCRIPTION
**Pull request by Martoon-00**

### CODE REVIEW

**Comment by Martoon-00** [comment]
WIP till https://github.com/input-output-hk/cardano-sl/pull/893 is merged

File: src/Pos/Wallet/Web/Api.hs [hunk 1]
```diff
@@ -108,103 +108,103 @@ type TestReset =
...
+type GetAccount =
        "wallets"
     :> CCapture "walletId" CAccountId
     :> WRes Get CAccount
 
-type GetWallets =
+type GetAccounts =
        "wallets"
-    :> QueryParam "walletSetId" (CId Wal)
+    :> QueryParam "walletId" (CId Wal)
```
Comment (akegalj) [inline src/Pos/Wallet/Web/Api.hs]: It has to reflect the change in BackendApi.purs https://github.com/input-output-hk/cardano-sl/blob/9411eb79c76a68b551620fe4574c7a257516684d/daedalus/src/Daedalus/BackendApi.purs#L140

File: src/Pos/Wallet/Web/Api.hs [hunk 2]
```diff
@@ -108,103 +108,103 @@ type TestReset =
...
     :> "sets"
     :> "keys"
     :> CQueryParam "passphrase" CPassPhrase
     :> ReqBody '[JSON] Text
     :> WRes Post CWallet
 
-type ChangeWalletSetPassphrase =
+type ChangeWalletPassphrase =
        "wallets"
     :> "sets"
```
Comment (akegalj) [inline src/Pos/Wallet/Web/Api.hs]: we should also change the rest api names. "sets" shouldn't exist. We should reflect this change in both Api.hs and BackendApi.purs

File: src/Pos/Wallet/Web/Api.hs [hunk 3]
```diff
@@ -108,103 +108,103 @@ type TestReset =
...
-    :> Capture "walletSetId" (CId Wal)
+    :> Capture "walletId" (CId Wal)
     :> Capture "name" Text
     :> WRes Post CWallet
 
-type DeleteWalletSet =
+type DeleteWallet =
        "wallets"
     :> "sets"
     :> Capture "walletSetId" (CId Wal)
```
Comment (akegalj) [inline src/Pos/Wallet/Web/Api.hs]: walletSetId again.

File: src/Pos/Wallet/Web/Api.hs [hunk 4]
```diff
@@ -365,39 +360,39 @@ type WalletApi = ApiPrefix :> (
...
+     DeleteWallet
     :<|>
-     ImportWalletSet
+     ImportWallet
     :<|>
-     ChangeWalletSetPassphrase
+     ChangeWalletPassphrase
     :<|>
      -------------------------------------------------------------------------
      -- Wallets
```
Comment (Pastafarianist) [inline src/Pos/Wallet/Web/Api.hs]: Do we need to update the comment as well?

File: src/Pos/Wallet/Web/Api.hs [hunk 5]
```diff
@@ -365,39 +360,39 @@ type WalletApi = ApiPrefix :> (
...
+     UpdateAccount
     :<|>
-     NewWallet
+     NewAccount
     :<|>
-     DeleteWallet
+     DeleteAccount
     :<|>
      -------------------------------------------------------------------------
      -- Accounts
```
Comment (Pastafarianist) [inline src/Pos/Wallet/Web/Api.hs]: Likewise, comment?

File: src/Pos/Wallet/Web/Api.hs (hunk 4 above)
Comment (Pastafarianist) [inline src/Pos/Wallet/Web/Api.hs]: Comment?

File: src/Pos/Wallet/Web/Api.hs (hunk 5 above)
Comment (Pastafarianist) [inline src/Pos/Wallet/Web/Api.hs]: Comment?

File: src/Pos/Wallet/Web/Api.hs [hunk 6]
```diff
@@ -365,39 +360,39 @@ type WalletApi = ApiPrefix :> (
      -------------------------------------------------------------------------
      -- Wallet sets
```
Comment (Pastafarianist) [inline src/Pos/Wallet/Web/Api.hs]: Do we need to update the comment as well?

**Comment by Martoon-00** [comment]
Oh lalki, I feel we have a chance to make this iteration the last one

File: src/Pos/Wallet/Web/Api.hs (hunk 5 above)
Comment (Martoon-00) [inline src/Pos/Wallet/Web/Api.hs]: Lil, where did you get this commit? ))
One which Ante commented before already has comment fixed

Comment (Martoon-00) [inline src/Pos/Wallet/Web/Api.hs]: Ah, I see, sorry )

**Comment by Martoon-00** [comment]
History-related endpoints are fixed by @Pastafarianist in #913

**Pull request merged**